build-evshim.ps1

```
@@ -0,0 +1,41 @@
+
+$SRC         = "app\src\main\cpp\extras\evshim.c"
+$INCLUDES    = "app\src\main\cpp\extras\sdl2_stub"
+$OUT         = "app\src\main\jniLibs\arm64-v8a\libevshim.so"
+
+if (-not (Test-Path $CLANG)) {
+    Write-Error "Clang not found at: $CLANG`nCheck that NDK $NDK_VERSION is installed."
+}
+
+Write-Host "Compiling evshim.c with NDK $NDK_VERSION ..."
+
+$args = @(
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)